tests/assets/hacks_for_tests.yaml

```diff
@@ -0,0 +1,14 @@
+name: "Hacks for testing environment"
```

Comment:
Downgrade tests downgrades TW to Leap 15.5 and this causes certain frictions, the gap is huge. Dbus services have been refactored and `dbus.service` in `/etc` is a symlink to `dbus-broker.service` in TW, however `dbus-broker.service` does not exist in Leap. Since `/etc/systemd` is persisted by default this is causing a glitch.

IMHO  `/etc/systemd` should have not been set as persistent by default #2226

Reply:
Changing entirely which distro you are running feels like a risky thing to try to support.. I would say if someone wants to change distros between upgrades the correct way would be to upgrade/downgrade the recovery to the wanted version and then do a reset which includes resetting persistent data..

That said I think this is a nice improvement and having more immutability in `/etc` is a good idea!